### PR TITLE
ANW-1505: Adding partial/whole extend in PUI

### PR DESCRIPTION
**Pull request by Vchouliaras**
## Description
Adds "partial/whole extend" literal on PUI collection overview page

## Related JIRA Ticket or GitHub Issue
https://archivesspace.atlassian.net/browse/ANW-1505

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have authority to submit this code.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Vchouliaras** [comment]
> Can we add a spec for this? I guess the proper place would be: `public/spec/features/record_innards_spec.rb`

I added them

**Review by thimios — APPROVED** [review]
@Vchouliaras please ensure that all changes are squashed into a single commit before merging.

Also please do not merge until @brianzelip has also reviewed this PR

**Review by brianzelip — APPROVED** [review]

**Pull request merged**
